integration-tests/docker-compose-configs/anchor-platform-default-configs/anchor-platform-config.yaml

```diff
@@ -1,128 +1,358 @@
+version: 1
```

Comment:
Is that possible for your docker compose to reuse `example.anchor-config.yaml`? Just wondering if we could get rid of this file.

Reply:
Good point! It'd be great to have that file tested in a CI flow but I understand that the docker setup makes it hard.